data/hosts.js

```diff
@@ -32,7 +32,11 @@ module.exports.hosts = [
 // ^^^ Add your ILP host by duplicating, uncommenting, and editing the line above ^^^
 
 // # hostname
-// This should have the format like 'example.com', so without the 'https://' in front.
+// This should have the format like 'example.com', so without the 'https://' in front, and the connectorland script
+// will do a WebFinger query for this host to find out which ledger(s) and which connector(s) are announced on there.
+// (if your connector is not represented by a DNS (sub)domain, you can add it in the list of "ILP-addressable
+// connectors" below)
```

Comment:
> if your connector is not represented by a DNS (sub)domain

I am wondering about the meaning of this sentence. A connector is not a server and is not listening for incoming connections, so there is no DNS domain linked to a connector. Proposal for rewrite: 

> If your connector is not announced in the webfinger response, you can add it ...

Edit: Further down you describe in more detail what to do if someone wants to add a connector that is not announced in the webfinger response. Maybe just drop the sentence from this paragraph.